### PR TITLE
Add a UWP project for issue 697

### PR DESCRIPTION
**Pull request by choenden**
This change adds a UWP project for OxyPlot with xamarin forms 2.0. It does not add this as a nuget package. Additionally, the sample project was expanded to include a uwp project.
Signed-off-by: Choden Konigsmark <choden.konigsmark@gmail.com>

### CODE REVIEW

**Comment by choenden** [comment]
I was not able to do the following two as part of the pull request - can someone tell me how? Should these be a separate commit into my fork?
Update the CHANGELOG.md file
Update the CONTRIBUTORS and AUTHORS files if you are not already listed there

**Comment by objorke** [comment]
Yes, the changes to the CHANGELOG.md and CONTRIBUTORS/AUTHORS should also be committed.
I am normally updating the change log in the same commit as the other files.

**Comment by objorke** [comment]
I think it is OK that you don't update the AppVeyor sln since Xamarin Forms UWP is in preview. When UWP is fully supported and #615 is completed, we should update the AppVeyor sln and create the NuGet package!

File: Source/OxyPlot.Xamarin.Forms.Platform.UWP/Forms.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Xamarin.Forms.Platform.UWP
+{
+    using System;
+
+    /// <summary>
+    /// Initializes OxyPlot renderers for use with Xamarin.Forms.
+    /// </summary>
+    public static class Forms
```
Comment (objorke) [inline Source/OxyPlot.Xamarin.Forms.Platform.UWP/Forms.cs]: I think this class and file can be removed.

Comment (choenden) [inline Source/OxyPlot.Xamarin.Forms.Platform.UWP/Forms.cs]: I thought it might be relevant for consistency with the wp8 project, but I removed it.

**Comment by choenden** [comment]
I am trying to merge the commits so that only one shows up.

**Comment by choenden** [comment]
Sorry for bugging you @objorke . I had to play around quite a bit because the deleted file kept coming back after git pull, which it kept asking me to do after git rebase -i for squashing.
I was finally able to push the squashed changes up, but it seems that change 4fa38033e722ade8d762b324afa54a078ab30917 from @tibel was accidentally squashed too. Is there anything I can do?

**Comment by choenden** [comment]
Ah, I think it has worked now

**Comment by choenden** [comment]
I wonder if something in the squash went wrong? This is the error from the appveyour build:
[00:02:15] INFO [12/31/15 0:39:48:62] Applicable build agent found: 'AppVeyor'.
[00:02:15] ERROR [12/31/15 0:40:05:89] An unexp
[00:02:15] ected error occurred:
[00:02:15] System.Net.WebException: The remote server returned an error: (500) Internal Server Error.
[00:02:15]    at System.Net.HttpWebRequest.GetResponse()
[00:02:15]    at GitVersion.AppVeyor.GenerateSetVersionMessage(VersionVariables variables)
[00:02:15]    at GitVersion.BuildServerBase.WriteIntegration(Action`1 writer, VersionVariables variables)
[00:02:15]    at GitVersion.SpecifiedArgumentRunner.Run(Arguments arguments, IFileSystem fileSystem)
[00:02:15]    at GitVersion.Program.VerifyArgumentsAndRun()
[00:02:15] INFO [12/31/15 0:40:05:89] 
[00:02:15] INFO [12/31/15 0:40:05:89] Here is the current git graph (please include in issue): 
[00:02:15] INFO [12/31/15 0:40:05:89] Showing max of 100 commits
[00:02:15] INFO [12/31/15 0:40:06:11] *   c955a22 3 minutes ago  (HEAD -> pull/700/merge)

**Comment by objorke** [comment]
I think the broken build is a GitVersion, I am starting a rebuild of the PR now.
The commit looks good to me. Will merge when the build succeeds.

**Comment by objorke** [comment]
GitVersion still crashes. I will try to merge anyway.

**Pull request merged**